cmd/prober/prober.go

```diff
@@ -104,44 +108,118 @@ func encodeLevel() zapcore.LevelEncoder {
 }
 
 var (
-	logStyle       string
-	frequency      int
+	scPath      string
+	trPath      string
+	trustedRoot *root.TrustedRoot
+	staging     bool
+
+	frequency int
+	logStyle  string
+	addr      string
+	grpcPort  int
+
 	retries        uint
-	addr           string
-	rekorURL       string
-	fulcioURL      string
-	fulcioGrpcURL  string
-	tsaURL         string
 	oneTime        bool
 	runWriteProber bool
-	versionInfo    version.Info
+
+	rekorV1Services []root.Service
```

Comment:
Can we declare these variables and `err` when we use them? Let's only declare global variables for the flag values.